docs/guides/hello.md

```diff
@@ -0,0 +1,1236 @@
+construction of a gRPC client-server application (it's the same example used in our [Quick Starts](/docs/quickstart)). It explains how gRPC works while showing you how to:
+
+- Create a protocol buffers schema that defines a simple RPC service with
+a single
+Hello World method.
+- Create a server that implements this interface in your favourite language (where available).
+- Create a client in your favourite language (or any other one you like!) that accesses your server.
+
+The complete code for the examples is available in the `examples` directory in our
+GitHub repositories. We use the Git versioning system for source code management:
+however, you don't need to know anything about Git to follow along other
+than how to install and run a few git commands.
+
+Note that server code for our example isn't available in all gRPC languages, as gRPC PHP and Objective-C only support creating clients.
+
+This is an introductory guide to gRPC rather than a comprehensive tutorial for any particular language. You can find more in-depth [tutorials](/docs/tutorials) and [reference information](/docs/reference) elsewhere in our site.
+
+
+## Setup
+
+<a name="setup"></a>
+This section explains how to set up your local machine to work with
+the example code. If you just want to read the examples, you can go straight
+to the [next step](#servicedef).
+
+### Install Git
+
+You can download and install Git from [http://git-scm.com/download](http://git-scm.com/download). Once
+installed you should have access to the git command line tool. The main
+commands that you will need to use are:
+
+- `git clone ...` : clone a remote repository onto your local machine
+- `git checkout ...` : check out a particular branch or a tagged version of
+the code to hack on
+
+
+### Get the source code
+
+<div class="tabs">
+  <ul>
+    <li><a href="#java_source">Java</a></li>
+    <li><a href="#cpp_source">C++</a></li>
+    <li><a href="#python_source">Python</a></li>
+    <li><a href="#go_source">Go</a></li>
+    <li><a href="#ruby_source">Ruby</a></li>
+    <li><a href="#node_source">Node.js</a></li>
+    <li><a href="#csharp_source">C#</a></li>
+    <li><a href="#objective-c_source">Objective-C</a></li>
+    <li><a href="#php_source">PHP</a></li>
+  </ul>
+  <div id="java_source">
+  <p>The example code for our Java example lives in the <code>grpc-java</code>
+  GitHub repository. Clone this repository to your local machine by running the
+  following command:</p>
+  <pre>
+  git clone https://github.com/grpc/grpc-java.git
+  </pre>
+  <p>Change your current directory to <code>grpc-java/examples</code>.</p>
+  <pre>
+  cd grpc-java/examples
+  </pre>
+  </div>
+
+  <div id="cpp_source">
+  <p>The example code for this lives in the <code>examples</code> directory in our GitHub repositories. Clone this repository to your local machine by running the following command:</p>
+  <pre>
+  $ git clone -b $(curl -L http://grpc.io/release) https://github.com/grpc/grpc
```

Comment:
these need to be changed to 
git clone -b {{site.data.config.grpc_release_branch}}  https://github.com/grpc/grpc 
/cc @dgquintas